UE/Application/Ports/IUserPort.hpp

```diff
@@ -21,13 +25,16 @@ class IUserPort
 public:
     virtual ~IUserPort() = default;
 
-    virtual void showNotConnected() = 0;
-    virtual void showConnecting() = 0;
-    virtual void showConnected() = 0;
-    virtual void showCallRequest(common::PhoneNumber)=0;
-    virtual void talk(common::PhoneNumber)=0;
-    virtual void showPartnerNotAvailable(common::PhoneNumber)=0;
-    virtual void showStartMenu()=0;
+    virtual void USER_showNotConnected() = 0;
+    virtual void USER_showConnecting() = 0;
+    virtual void USER_showConnected() = 0;
+    virtual void USER_showCallRequest(common::PhoneNumber)=0;
```

Comment:
Dlaczego znowu mieszasz style spacji? Jak je usuniesz to nie oszczędzisz na miejscu, a stracisz na czytelności